policy-controller/k8s/status/src/index.rs

```diff
@@ -285,25 +287,29 @@ impl Index {
     fn make_http_route_patch(
         &self,
         id: &NamespaceGroupKindName,
-        parents: &[ParentReference],
-        backends: &[BackendReference],
+        route: &HttpRoute,
     ) -> k8s::Patch<serde_json::Value> {
-        let backend_condition = self.backend_condition(backends);
-        let parent_statuses = parents
+        let backend_condition = self.backend_condition(&route.backends);
+        let unowned_statuses = route
+            .statuses
             .iter()
-            .filter_map(|parent_ref| self.parent_status(parent_ref, backend_condition.clone()))
-            .collect();
+            .filter(|status| status.controller_name != POLICY_CONTROLLER_NAME)
+            .cloned();
+        let parent_statuses = route
+            .parents
+            .iter()
+            .filter_map(|parent_ref| self.parent_status(parent_ref, backend_condition.clone()));
```

Comment:
Can you help me understand this by adding a comment that explains why this is load bearing?

Is this essentially that we are "replacing" any statuses where the controller name is ours? I think the implicit bit here is that parent_status only returns the status of things for which we control status?